modules/client/src/interfaces.ts

```diff
@@ -539,26 +536,31 @@ export type ContractWithdrawParams = [string, string, BigNumber, string];
 
 export type PrepareUninstallationParams = {
   daoAddressOrEns: string;
-  versionTag: VersionTag;
   pluginAddress: string;
   pluginInstallationIndex?: number;
+  uninstallationParams?: any;
 };
 
 export type SubgraphPluginVersion = {
   release: {
     release: number;
   };
+  metadata: string;
```

Comment:
Which metadata? The release or the build's?

Reply:
build's you want me to specify it?

Reply:
oh wait, that's what comes from subgraph
Adding a comment like `/** build metadata URI */`  will help us here

Reply:
👍